src/ai/backend/manager/api/session.py

```diff
@@ -407,18 +407,16 @@ async def _create(request: web.Request, params: Any) -> web.Response:
         if len(alias_folders) != len(set(alias_folders)):
             raise InvalidAPIParameters('Duplicate alias folder name exists.')
 
-        p: str
-        for p in alias_folders:
-            if p is None:
+        alias_name: str
+        for alias_name in alias_folders:
+            alias_name = alias_name.replace('/', '')
+            if alias_name is None:
                 continue
-            if not p.startswith('/home/work/'):
-                raise InvalidAPIParameters(f'Path {p} should start with /home/work/')
-
-            alias_name = p.replace('/home/work/', '')
             if alias_name == '':
                 raise InvalidAPIParameters('Alias name cannot be empty.')
             if not verify_vfolder_name(alias_name):
-                raise InvalidAPIParameters(f'Path {str(p)} is reserved for internal operations.')
+                raise InvalidAPIParameters(f'Path {str(alias_name)} is reserved for \
+                                             internal operations.')
```

Comment:
Indentation suggestion:
```suggestion
                raise InvalidAPIParameters(
                    f'Path {str(alias_name)} is reserved for internal operations.',
                )
```
Since this part of code has been written long ago, we don't use this style any more.
And even with the style to match indents with the beginning parenthesis, `f'` shouldn't be included in the indentation of following lines.